client/src/udp_client.rs

```diff
@@ -39,4 +39,12 @@ impl TpuConnection for UdpTpuConnection {
             })?;
         Ok(())
     }
+
+    fn send_wire_transaction_batch(&self, buffers: &[&[u8]]) -> TransportResult<()> {
+        buffers.iter().try_for_each(|buff| -> TransportResult<()> {
+            self.socket.send_to(buff, self.addr)?;
```

Comment:
use send_mmsg here?

Reply:
Have changed the UDP implementation to use send_mmsg